### PR TITLE
Update file-attachments-on-issues-and-pull-requests.md

### PR DESCRIPTION
**Pull request by daminebenq**
### Why:

Now Github supports mp4 file attachments. So i saw that it is missing in this list.

### What's being changed:

https://user-images.githubusercontent.com/5144311/107952135-21471f80-6f78-11eb-927e-796940913815.mp4

### Check off the following:
- [x] I have reviewed my changes in staging. (look for the **deploy-to-heroku** link in your pull request, then click **View deployment**)
- [x] For content changes, I have reviewed the [localization checklist](https://github.com/github/docs/blob/main/contributing/localization-checklist.md)
- [x] For content changes, I have reviewed the [Content style guide for GitHub Docs](https://github.com/github/docs/blob/main/contributing/content-style-guide.md).

### CODE REVIEW

File: content/github/managing-your-work-on-github/file-attachments-on-issues-and-pull-requests.md [hunk 1]
```diff
@@ -31,6 +31,7 @@ We support these files:
 
 * PNG (*.png*)
 * GIF (*.gif*)
+* MP4 (*.mp4*)
```
Comment (herackovalucka) [inline content/github/managing-your-work-on-github/file-attachments-on-issues-and-pull-requests.md:34]: Np

Comment (daminebenq) [reply content/github/managing-your-work-on-github/file-attachments-on-issues-and-pull-requests.md:34]: So ? Is this an approval ?

Comment (daminebenq) [reply content/github/managing-your-work-on-github/file-attachments-on-issues-and-pull-requests.md:34]: So ? Is this an approval ?

**Comment by janiceilene** [comment]
@daminebenq Thanks so much for opening a PR! I'll get this triaged for review ⚡

**Comment by felicitymay** [comment]
👋🏻 Hi @daminebenq - thanks for taking the time to open this PR. 

You're right to say that GitHub now supports mp4 attachments. It was announced as a beta feature: https://github.blog/changelog/2020-12-16-video-upload-public-beta/ and https://github.blog/changelog/2021-02-01-expanded-support-for-video-uploads/.

It looks as if someone else has also opened a PR to update to this page. Since that PR also adds the limit on file sizes, I'm closing your PR in favor of: https://github.com/github/docs/pull/3571.

**Pull request closed without merging**